src/MOI_wrapper.jl

```diff
@@ -2640,129 +2628,96 @@ function _is_feasible(model::Optimizer)
     return model.inner.conflict == Gurobi.GRB_INFEASIBLE
 end
 
-"""
-    ConflictStatus()
-
-Return an `MOI.TerminationStatusCode` indicating the status of the last
-computed conflict. If a minimal conflict is found, it will return
-`MOI.OPTIMAL`. If the problem is feasible, it will return `MOI.INFEASIBLE`. If
-`compute_conflict` has not been called yet, it will return
-`MOI.OPTIMIZE_NOT_CALLED`.
-"""
-struct ConflictStatus <: MOI.AbstractModelAttribute end
```

Comment:
Could you add `@deprecate ConflictStatus MOI.ConflictStatus` ?

Reply:
Even better would be to keep `Gurobi.ConflictStatus` and let it return the solver-specific `Gurobi.GRB_...` and call this from the getter of `MOI.ConflictStatus`. That would allow the user to have the solver-specific status in case he wants more details.

Reply:
Isn't defining the same attribute for several solvers defeating the purpose of MOI? Wouldn't it be better to have something like `MOI.RawConflictStatus`?

Reply:
The purpose of MOI is to have solver-independent and solver-dependent features coexist seemlessly.
Here, it's not more work to add a Gurobi-specific attribute that could be accessed even at the JuMP level.
We could have a `RawConflictStatus` but before we do, we could have solver-specific attributes.
The situation is similar for callbacks, we have both solver-independent ones and solver-dependent ones.

Reply:
Here it is (with tests included)!